development/language/validation.rst

```diff
@@ -83,19 +81,95 @@ File Validation
   before ``<?php``. ``.php`` files must **not** contain the closing tag ``?>``,
   but just end with a new line
 
-language/{iso}/iso.txt
-----------------------
+language/{iso}/composer.json
+----------------------------
+It is the main configuration file of your translation and language packages.
 
-* The file must contain exactly 3 lines:
+The ``composer.json`` from the default language `British English` looks like this:
 
-    #. English language name
-    #. Language name in the spoken language
-    #. Author(-group) information (Plaintext only, no links allowed)
+.. code-block:: json
+
+        {
+            "name": "phpbb/phpbb-language-en",
+            "description": "phpBB Forum Software default language",
+            "type": "phpbb-language",
+            "version": "4.0.0-a1-dev",
+            "homepage": "https://www.phpbb.com",
+            "license": "GPL-2.0",
+            "authors": [
+                {
+                    "name": "phpBB Limited",
+                    "email": "operations@phpbb.com",
+                    "homepage": "https://www.phpbb.com/go/authors"
+                }
+            ],
+            "support": {
+                "issues": "https://tracker.phpbb.com",
+                "forum": "https://www.phpbb.com/community/",
+                "wiki": "https://wiki.phpbb.com",
+                "irc": "irc://irc.freenode.org/phpbb"
+            },
+            "extra": {
+                "language-iso": "en",
+                "english-name": "British English",
+                "local-name": "British English",
+                "phpbb-version": "4.0.0-a1-dev",
+                "direction": "ltr",
+                "user-lang": "en-gb",
+                "plural-rule": 1,
+                "recaptcha-lang": "en-GB"
+            }
+        }
+
+Main block
+^^^^^^^^^^
+The main part of this ``composer.json`` has six different types of information:
+
+* ``"name":`` need to start with ``phpbb/phpbb-language-`` and followed by the language iso code e.g. ``phpbb/phpbb-language-de``
+* ``"description":`` should contain a short description for your translation e.g. ``phpBB Forum Software language package Dutch (Casual Honorifics)``
+* ``"type":`` must be: ``"phpbb-language",``. Just leave it as it is!
+* ``"version":`` should be the version number of the language package. This can be different then the phpBB-version it is made for.
+* ``"homepage":`` if you want, you can put an url to your website. If not, left it empty.
+* ``"license":`` must be: ``"GPL-2.0",``. Just leave it as it is!
+
+Authors
+^^^^^^^
+Allows you to mention the organisation or people who did and maintain the translation. You can just add repeating blocks like this to add more than one person:
```

Comment:
Rephrase:
Use this section to credit the authors and maintainers of this translation. You can add repeating blocks like this for each additional author.